shared/chat/conversation/info-panel/settings/index.tsx

```diff
@@ -83,6 +84,89 @@ const SettingsPanel = (props: SettingsPanelProps) => {
 
   const onLeaveConversation = () => dispatch(Chat2Gen.createLeaveConversation({conversationIDKey}))
 
+  if (flags.teamsRedesign) {
+    return (
+      <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.settingsContainer}>
+        <Kb.ScrollView>
+          {isPreview ? (
+            <Kb.Box2 direction="vertical" fullWidth={true} style={styles.settingsHeader}>
+              <Kb.Text type="BodySmallSemibold">You are not in this channel.</Kb.Text>
+              <Kb.Button mode="Secondary" label="Join channel" style={styles.buttonStyle} />
+            </Kb.Box2>
+          ) : (
+            <Notifications conversationIDKey={conversationIDKey} />
+          )}
+
+          {entityType === 'channel' && channelname !== 'general' && !isPreview && (
+            <Kb.Button
+              type="Danger"
+              mode="Secondary"
+              label="Leave channel"
+              onClick={onLeaveConversation}
+              style={styles.smallButton}
+            />
+          )}
+
+          <Kb.Box2 direction="vertical" fullWidth={true} style={styles.settingsHeader}>
```

Comment:
why have a box around this text? you can style the text directly if you just need the styles.